spec/unit/outputs/elasticsearch/http_client/manticore_adapter_spec.rb

```diff
@@ -44,19 +44,35 @@
     end
   end
 
+  describe "bad response codes" do
+    let(:uri) { ::LogStash::Util::SafeURI.new("http://localhost:9200") }
+
+    it "should raise a bad response code error" do
```

Comment:
I may be reading this wrong, but I don't see a point to this test, since we mock the response, stub it, and then check we got the values from the mock.
Also, this test does not break in master.